# Translations template for neutron.
# Copyright (C) 2016 ORGANIZATION
# This file is distributed under the same license as the neutron project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: neutron 8.0.1.dev89\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2016-04-08 06:35+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.2.0\n"

#: neutron/policy.py:115
#, python-format
msgid "Unable to find data type descriptor for attribute %s"
msgstr ""

#: neutron/agent/rpc.py:121
msgid "DVR functionality requires a server upgrade."
msgstr ""

#: neutron/agent/rpc.py:199
msgid "Tunnel synchronization requires a server upgrade."
msgstr ""

#: neutron/agent/securitygroups_rpc.py:66
#: neutron/agent/securitygroups_rpc.py:100
msgid "Driver configuration doesn't match with enable_security_group"
msgstr ""

#: neutron/agent/securitygroups_rpc.py:130
msgid ""
"security_group_info_for_devices rpc call not supported by the server, "
"falling back to old security_group_rules_for_devices which scales worse."
msgstr ""

#: neutron/agent/common/ovs_lib.py:452
#, python-format
msgid "Found not yet ready openvswitch port: %s"
msgstr ""

#: neutron/agent/common/ovs_lib.py:455
#, python-format
msgid "Found failed openvswitch port: %s"
msgstr ""

#: neutron/agent/common/ovs_lib.py:514
#, python-format
msgid "ofport: %(ofport)s for VIF: %(vif)s is not a positive integer"
msgstr ""

#: neutron/agent/dhcp/agent.py:117
#, python-format
msgid ""
"Unable to %(action)s dhcp for %(net_id)s: there is a conflict with its "
"current state; please check that the network and/or its subnet(s) still "
"exist."
msgstr ""

#: neutron/agent/dhcp/agent.py:226
#, python-format
msgid ""
"Network %s may have been deleted and its resources may have already been "
"disposed."
msgstr ""

#: neutron/agent/dhcp/agent.py:383
#, python-format
msgid ""
"%(port_num)d router ports found on the metadata access network. Only the "
"port %(port_id)s, for router %(router_id)s will be considered"
msgstr ""

#: neutron/agent/dhcp/agent.py:589 neutron/agent/l3/agent.py:686
#: neutron/agent/metadata/agent.py:270
#: neutron/services/bgp/agent/bgp_dragent.py:689
#: neutron/services/metering/agents/metering_agent.py:279
msgid ""
"Neutron server does not support state report. State report for this agent"
" will be disabled."
msgstr ""

#: neutron/agent/l2/extensions/qos.py:105
#, python-format
msgid "Unsupported QoS rule type for %(rule_id)s: %(rule_type)s; skipping"
msgstr ""

#: neutron/agent/l3/agent.py:216
#, python-format
msgid ""
"l3-agent cannot check service plugins enabled at the neutron server when "
"startup due to RPC error. It happens when the server does not support "
"this RPC API. If the error is UnsupportedVersion you can ignore this "
"warning. Detail message: %s"
msgstr ""

#: neutron/agent/l3/agent.py:228
#, python-format
msgid ""
"l3-agent cannot check service plugins enabled on the neutron server. "
"Retrying. Detail message: %s"
msgstr ""

#: neutron/agent/l3/agent.py:365
#, python-format
msgid "Info for router %s was not found. Performing router cleanup"
msgstr ""

#: neutron/agent/l3/dvr_local_router.py:197
#, python-format
msgid ""
"Device %s does not exist so ARP entry cannot be updated, will cache "
"information to be applied later when the device exists"
msgstr ""

#: neutron/agent/l3/router_info.py:268
#, python-format
msgid "Unable to configure IP address for floating IP: %s"
msgstr ""

#: neutron/agent/linux/dhcp.py:233
#, python-format
msgid "Failed trying to delete interface: %s"
msgstr ""

#: neutron/agent/linux/dhcp.py:240
#, python-format
msgid "Failed trying to delete namespace: %s"
msgstr ""

#: neutron/agent/linux/external_process.py:250
#, python-format
msgid "Respawning %(service)s for uuid %(uuid)s"
msgstr ""

#: neutron/agent/linux/interface.py:342 neutron/agent/linux/interface.py:409
#: neutron/agent/linux/interface.py:456
#, python-format
msgid "No MTU configured for port %s"
msgstr ""

#: neutron/agent/linux/iptables_manager.py:253
#, python-format
msgid ""
"Tried to remove rule that was not there: %(chain)r %(rule)r %(wrap)r "
"%(top)r"
msgstr ""

#: neutron/agent/linux/iptables_manager.py:660
#, python-format
msgid ""
"Duplicate iptables %(thing)s detected. This may indicate a bug in the the"
" iptables %(thing)s generation code. Line: %(line)s"
msgstr ""

#: neutron/agent/linux/iptables_manager.py:699
#, python-format
msgid "Attempted to get traffic counters of chain %s which does not exist"
msgstr ""

#: neutron/agent/metadata/agent.py:203
msgid ""
"The remote metadata server responded with Forbidden. This response "
"usually occurs when shared secrets do not match."
msgstr ""

#: neutron/api/api_common.py:110
#, python-format
msgid ""
"Invalid value for pagination_max_limit: %s. It should be an integer "
"greater to 0"
msgstr ""

#: neutron/api/extensions.py:537
#, python-format
msgid "Did not find expected name \"%(ext_name)s\" in %(file)s"
msgstr ""

#: neutron/api/extensions.py:545
#, python-format
msgid "Extension file %(f)s wasn't loaded due to %(exception)s"
msgstr ""

#: neutron/api/extensions.py:583
#, python-format
msgid "Extension %s not supported by any of loaded plugins"
msgstr ""

#: neutron/api/extensions.py:594
#, python-format
msgid "Loaded plugins do not implement extension %s interface"
msgstr ""

#: neutron/api/rpc/agentnotifiers/dhcp_rpc_agent_api.py:81
#, python-format
msgid ""
"Unable to schedule network %s: no agents available; will retry on "
"subsequent port and subnet creation events."
msgstr ""

#: neutron/api/rpc/agentnotifiers/dhcp_rpc_agent_api.py:99
#, python-format
msgid ""
"Only %(active)d of %(total)d DHCP agents associated with network "
"'%(net_id)s' are marked as active, so notifications may be sent to "
"inactive agents."
msgstr ""

#: neutron/api/rpc/handlers/dhcp_rpc.py:107
#, python-format
msgid ""
"Action %(action)s for network %(net_id)s could not complete successfully:"
" %(reason)s"
msgstr ""

#: neutron/api/rpc/handlers/dhcp_rpc.py:187
#, python-format
msgid "Updating lease expiration is now deprecated. Issued  from host %s."
msgstr ""

#: neutron/api/rpc/handlers/securitygroups_rpc.py:188
msgid ""
"Security group agent binding currently not set. This should be set by the"
" end of the init process."
msgstr ""

#: neutron/cmd/sanity_check.py:94
msgid ""
"The user that is executing neutron can read the namespaces without using "
"the root_helper. Disable the use_helper_for_ns_read option to avoid a "
"performance impact."
msgstr ""

#: neutron/db/agents_db.py:202
#, python-format
msgid "%(agent_type)s agent %(agent_id)s is not active"
msgstr ""

#: neutron/db/agents_db.py:224
#, python-format
msgid ""
"Dictionary %(dict_name)s for agent %(agent_type)s on host %(host)s is "
"invalid."
msgstr ""

#: neutron/db/agents_db.py:290
#, python-format
msgid ""
"Agent healthcheck: found %(count)s dead agents out of %(total)s:\n"
"%(data)s"
msgstr ""

#: neutron/db/agentschedulers_db.py:146
#, python-format
msgid ""
"Time since last %s agent reschedule check has exceeded the interval "
"between checks. Waiting before check to allow agents to send a heartbeat "
"in case there was a clock adjustment."
msgstr ""

#: neutron/db/agentschedulers_db.py:285
msgid "No DHCP agents available, skipping rescheduling"
msgstr ""

#: neutron/db/agentschedulers_db.py:289
#, python-format
msgid ""
"Removing network %(network)s from agent %(agent)s because the agent did "
"not report to the server in the last %(dead_time)s seconds."
msgstr ""

#: neutron/db/bgp_dragentscheduler_db.py:72
#: neutron/db/bgp_dragentscheduler_db.py:84
msgid "Cannot schedule BgpSpeaker to DrAgent. Reason: No scheduler registered."
msgstr ""

#: neutron/db/l3_agentschedulers_db.py:124
#, python-format
msgid ""
"Rescheduling router %(router)s from agent %(agent)s because the agent did"
" not report to the server in the last %(dead_time)s seconds."
msgstr ""

#: neutron/db/l3_agentschedulers_db.py:319
#, python-format
msgid ""
"Failed to notify L3 agent on host %(host)s about added router. Attempt "
"%(attempt)d out of %(max_attempts)d"
msgstr ""

#: neutron/db/l3_dvr_db.py:848
#, python-format
msgid "Router %s was not found. Skipping agent notification."
msgstr ""

#: neutron/db/securitygroups_rpc_base.py:378
#, python-format
msgid "No valid gateway port on subnet %s is found for IPv6 RA"
msgstr ""

#: neutron/debug/debug_agent.py:111
#, python-format
msgid "Failed to delete namespace %s"
msgstr ""

#: neutron/notifiers/nova.py:164
msgid "Port ID not set! Nova will not be notified of port status change."
msgstr ""

#: neutron/notifiers/nova.py:232
#, python-format
msgid "Nova event: %s returned with failed status"
msgstr ""

#: neutron/pecan_wsgi/startup.py:55
#, python-format
msgid "No plugin found for: %s"
msgstr ""

#: neutron/pecan_wsgi/startup.py:130
#, python-format
msgid "No plugin found for resource:%s. API calls may not be correctly dispatched"
msgstr ""

#: neutron/pecan_wsgi/controllers/resource.py:74
#: neutron/pecan_wsgi/controllers/root.py:98
#, python-format
msgid "No controller found for: %s - returning response code 404"
msgstr ""

#: neutron/plugins/hyperv/agent/security_groups_driver.py:27
#, python-format
msgid ""
"You are using the deprecated firewall driver: %(deprecated)s.Use the "
"recommended driver %(new)s instead."
msgstr ""

#: neutron/plugins/ml2/driver_context.py:198
#, python-format
msgid "Could not expand segment %s"
msgstr ""

#: neutron/plugins/ml2/managers.py:316
msgid ""
"Host filtering is disabled because at least one mechanism doesn't support"
" it."
msgstr ""

#: neutron/plugins/ml2/managers.py:360
#, python-format
msgid "%s does not support QoS; no rule types available"
msgstr ""

#: neutron/plugins/ml2/managers.py:741
#, python-format
msgid "Failed to bind port %(port)s on host %(host)s at level %(lvl)s"
msgstr ""

#: neutron/plugins/ml2/plugin.py:862
msgid "A concurrent port creation has occurred"
msgstr ""

#: neutron/plugins/ml2/plugin.py:1552
#, python-format
msgid "Port %s not found during update"
msgstr ""

#: neutron/plugins/ml2/rpc.py:93
#, python-format
msgid ""
"Device %(device)s requested by agent %(agent_id)s on network "
"%(network_id)s not bound, vif_type: %(vif_type)s"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_agent.py:69
#, python-format
msgid ""
"Port %(pid)s on network %(network)s not bound, no agent registered on "
"host %(host)s"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_agent.py:83
#, python-format
msgid "Refusing to bind port %(pid)s to dead agent: %(agent)s"
msgstr ""

#: neutron/plugins/ml2/drivers/type_flat.py:138
#, python-format
msgid "No flat network found on physical network %s"
msgstr ""

#: neutron/plugins/ml2/drivers/type_tunnel.py:238
#, python-format
msgid "%(type)s tunnel %(id)s not found"
msgstr ""

#: neutron/plugins/ml2/drivers/type_tunnel.py:304
#, python-format
msgid "Endpoint with ip %s already exists"
msgstr ""

#: neutron/plugins/ml2/drivers/type_vlan.py:257
#, python-format
msgid "No vlan_id %(vlan_id)s found on physical network %(physical_network)s"
msgstr ""

#: neutron/plugins/ml2/drivers/l2pop/mech_driver.py:113
#, python-format
msgid "unable to modify mac_address of ACTIVE port %s"
msgstr ""

#: neutron/plugins/ml2/drivers/l2pop/mech_driver.py:209
#, python-format
msgid "Unable to retrieve active L2 agent on host %s"
msgstr ""

#: neutron/plugins/ml2/drivers/linuxbridge/agent/linuxbridge_neutron_agent.py:136
msgid "Invalid Network ID, will lead to incorrect bridge name"
msgstr ""

#: neutron/plugins/ml2/drivers/linuxbridge/agent/linuxbridge_neutron_agent.py:144
msgid "Invalid VLAN ID, will lead to incorrect subinterface name"
msgstr ""

#: neutron/plugins/ml2/drivers/linuxbridge/agent/linuxbridge_neutron_agent.py:176
msgid "Invalid Interface ID, will lead to incorrect tap device name"
msgstr ""

#: neutron/plugins/ml2/drivers/linuxbridge/agent/linuxbridge_neutron_agent.py:186
#, python-format
msgid "Invalid Segmentation ID: %s, will lead to incorrect vxlan device name"
msgstr ""

#: neutron/plugins/ml2/drivers/linuxbridge/agent/linuxbridge_neutron_agent.py:605
#: neutron/plugins/ml2/drivers/linuxbridge/agent/linuxbridge_neutron_agent.py:642
#, python-format
msgid ""
"Option \"%(option)s\" must be supported by command \"%(command)s\" to "
"enable %(mode)s mode"
msgstr ""

#: neutron/plugins/ml2/drivers/linuxbridge/agent/linuxbridge_neutron_agent.py:636
msgid ""
"VXLAN muticast group(s) must be provided in vxlan_group option to enable "
"VXLAN MCAST mode"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_sriov/agent/eswitch_manager.py:208
#, python-format
msgid "Cannot find vf index for pci slot %s"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_sriov/agent/eswitch_manager.py:365
#, python-format
msgid "device pci mismatch: %(device_mac)s - %(pci_slot)s"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_sriov/agent/eswitch_manager.py:389
#, python-format
msgid ""
"VF with PCI slot %(pci_slot)s is already assigned; skipping reset maximum"
" rate"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_sriov/agent/pci_lib.py:166
#, python-format
msgid "Cannot find vfs %(vfs)s in device %(dev_name)s"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_sriov/agent/pci_lib.py:182
#, python-format
msgid "failed to parse vf link show line %(line)s: for %(device)s"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_sriov/agent/sriov_nic_agent.py:236
#, python-format
msgid "Failed to set spoofcheck for device %s"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_sriov/agent/sriov_nic_agent.py:245
#, python-format
msgid "Device %s does not support state change"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_sriov/agent/sriov_nic_agent.py:248
#, python-format
msgid "Failed to set device %s state"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_sriov/agent/sriov_nic_agent.py:329
#, python-format
msgid "port_id to device with MAC %s not found"
msgstr ""

#: neutron/plugins/ml2/drivers/mech_sriov/mech_driver/mech_driver.py:130
#, python-format
msgid "Attempting to bind with dead agent: %s"
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_dvr_neutron_agent.py:193
#, python-format
msgid ""
"L2 agent could not get DVR MAC address from server. Retrying. Detailed "
"message: %s"
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_dvr_neutron_agent.py:380
#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_dvr_neutron_agent.py:533
#, python-format
msgid ""
"DVR: Unable to retrieve subnet information for subnet_id %s. The subnet "
"or the gateway may have already been deleted"
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_neutron_agent.py:589
#, python-format
msgid "Action %s not supported"
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_neutron_agent.py:1384
#, python-format
msgid "VIF port: %s has no ofport configured, and might not be able to transmit"
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_neutron_agent.py:1509
#, python-format
msgid "Device %s not defined on plugin or binding failed"
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_neutron_agent.py:1665
#, python-format
msgid "Invalid remote IP: %s"
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_neutron_agent.py:1718
msgid "OVS is restarted. OVSNeutronAgent will reset bridges and recover ports."
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_neutron_agent.py:1721
msgid ""
"OVS is dead. OVSNeutronAgent will keep running and checking OVS status "
"periodically."
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_neutron_agent.py:1778
#, python-format
msgid "Clearing cache of registered ports, retries to resync were > %s"
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/ovs_neutron_agent.py:1852
#, python-format
msgid "Device %(dev)s failed for %(times)s times and won't be retried anymore"
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/openflow/native/ofswitch.py:141
#, python-format
msgid "Deleting flow with cookie 0x%(cookie)x"
msgstr ""

#: neutron/plugins/ml2/drivers/openvswitch/agent/openflow/ovs_ofctl/ofswitch.py:110
#, python-format
msgid "Deleting flow %s"
msgstr ""

#: neutron/quota/__init__.py:214
msgid ""
"The quota driver neutron.quota.ConfDriver is deprecated as of Liberty. "
"neutron.db.quota.driver.DbQuotaDriver should be used in its place"
msgstr ""

#: neutron/quota/resource.py:309 neutron/services/timestamp/timestamp_db.py:90
#, python-format
msgid "No sqlalchemy event for resource %s found"
msgstr ""

#: neutron/quota/resource_registry.py:215
#, python-format
msgid "%s is already registered"
msgstr ""

#: neutron/scheduler/dhcp_agent_scheduler.py:63
#, python-format
msgid "DHCP agent %s is not active"
msgstr ""

#: neutron/scheduler/dhcp_agent_scheduler.py:210
msgid "No more DHCP agents"
msgstr ""

#: neutron/scheduler/l3_agent_scheduler.py:149
#, python-format
msgid "No routers compatible with L3 agent configuration on host %s"
msgstr ""

#: neutron/scheduler/l3_agent_scheduler.py:174
msgid "No active L3 agents"
msgstr ""

#: neutron/scheduler/l3_agent_scheduler.py:180
#, python-format
msgid "No L3 agents can host the router %s"
msgstr ""

#: neutron/services/provider_configuration.py:125
#, python-format
msgid ""
"The configured driver %(driver)s has been moved, automatically using "
"%(new_driver)s instead. Please update your config files, as this "
"automatic fixup will be removed in a future release."
msgstr ""

#: neutron/services/bgp/agent/bgp_dragent.py:306
#, python-format
msgid "BGP Speaker %s has been deleted."
msgstr ""

#: neutron/services/bgp/agent/bgp_dragent.py:321
#, python-format
msgid "BGP Peer %s has been deleted."
msgstr ""

#: neutron/services/bgp/agent/bgp_dragent.py:335
#, python-format
msgid ""
"BGP speaker %s may have been deleted and its resources may have already "
"been disposed."
msgstr ""

#: neutron/services/bgp/scheduler/bgp_dragent_scheduler.py:143
#, python-format
msgid "BgpDrAgent %s is down"
msgstr ""

#: neutron/services/qos/notification_drivers/message_queue.py:30
#, python-format
msgid "Received %(resource)s %(policy_id)s without context"
msgstr ""

#: neutron/services/timestamp/timestamp_db.py:52
msgid ""
"The input changed_since must be in the following format: YYYY-MM-"
"DDTHH:MM:SS"
msgstr ""

